### DISCUSSION TITLE
Integrate CISAs Known Exploited Vulnerabilities Catalog?

### DISCUSSION THREAD

**Discussion by jbruenner** [DependencyTrack/dependency-track#2020] [Q&A] [answered]
My use case would be: Patching everything with EPSS > 0.5 and CVSS > 5.
This would work if known exploited vulnerabilities would fit in. They don't.
There are exploited vulnerabilities, having EPSS < 0.5.

I tried to find a way putting the known exploited vulnerabilities into DT.
Unfortunatly I failed. Is there any way to do it within DT?

Thanks a lot for your help
Jörg

**Comment by wolf-tme-ssg** [comment, marked as answer]
Hello @jbruenner,

DT currently does not integrate [CISA's known exploited vulnerabilities catalog](https://www.cisa.gov/known-exploited-vulnerabilities-catalog) and you cannot manually add it nor update EPSS scores which are automatically downloaded from [FIRST API](https://api.first.org/data/v1/epss).
I suggest that you create an enhancement request with the feature and expected user experience on GUI.
